Standardise the issue template for JupyterHub repositories (#1)
jupyterhub/.github: .github/ISSUE_TEMPLATE/feature_request.md

---
name: "\U0001F680 Feature request"
about: Suggest a new feature or a big change

---
<!-- Thank you for contributing. These HTML commments will not render in the issue, but you can delete them once you've read them if you prefer! -->

### Proposed change
<!-- Use this section to describe the feature you'd like to be added. -->

### Alternative options
<!-- Use this section to describe alternative options and why you've decided on the proposed feature above. -->

### Who would use this feature?

> <!-- Describe the audience for this feature. This information will affect who chooses to work on the feature with you. -->

Reviewer: ```suggestion
<!-- Describe who would benefit from using this feature. -->
```

Reviewer: I've considered part `who chooses to work on the feature with you.`, it implies it will be a collaboration and the person suggesting the feature will be involved doing so.

I like the idea of having these as slimmed as possible, and I think it has some value to communicate the value information could have, but we cannot communicate everything so I figured perhaps we should slim this down a bit.

> ### (Optional): Suggest a solution
> <!-- Describe what you think needs to be done. This information will affect who chooses to work on the feature with you -->

Reviewer: ```suggestion
### (Optional): Suggest a solution
<!-- Describe what you think needs to be done. Doing that is an excellent first step to get the feature implemented. -->
```

> ### How much effort will adding it take?
> <!-- Try to estimate how much work adding this feature will require. This information will affect who chooses to work on the feature with you. -->

Reviewer: ```suggestion
### Implementation ideas
<!-- Describe what you think needs to be done. -->
```

Reviewer: I figure asking for effort required requires some understanding of what needs to be done, asking for a description of that may help us become more actionable and could be the first step towards an indication from a maintainer that a PR doing this would be appreciated.

I'm not confident about my formulation, but I like having a place for the person submitting the feature request to elaborate on what they think may need to be done.

Author: I agree that evaluating the effort might be too much in the early stages of an idea. @consideRatio, what do you think about mentioning that offering a solution it's optional and merging the two versions into something like this?
```suggestion
### (Optional): Suggest a solution
<!-- Describe what you think needs to be done. This information will affect who chooses to work on the feature with you -->
```

> 
> 
> ### Who can do this work?
> <!-- What skills are needed? Who can be recruited to add this feature? This information will affect who chooses to work on the feature with you. -->

Reviewer: ```suggestion
```

I think the _who_ part is problematic unless the response is "I can do it". I think the _what skills_ part could be relevant, but I think it can follow from the suggest a solution part above.
